packages/in_app_purchase/in_app_purchase_android/lib/src/types/google_play_purchase_details.dart

```diff
@@ -19,31 +19,21 @@ class GooglePlayPurchaseDetails extends PurchaseDetails {
     required String? transactionDate,
     required this.billingClientPurchase,
     required PurchaseStatus status,
+    bool pendingCompletePurchase = false,
```

Comment:
This looks like a public API change? The minor version should be updated.

Reply:
Can modify it to be a non-public-API change if my below comment is applied.

Reply:
Done, thank you for the suggestion.